### ISSUE TITLE
Tamper-proof gps marker on records

### ISSUE THREAD

**Issue by japonicus** [NERC-CEH/irecord-app#46]
Tangentially related to Tom A's bug #45 is there a need for the 'location source: gps' flag to update if the record is edited afterwards in iRecord.  As it stands I can edit in a different, typed-in grid-reference on the iRecord website, yet the record still has the gps tag. Possibly the resulting conflict with the Gridref_calculated field is sufficient - but there could be scope for confusion.

For GPS'ed references to be trusted the system needs to be absolutely clear that they come from a GPS rather than from a later edit.

**Comment by kazlauskis** [comment]
I agree, this is an issue. Do you think we actually need to display _location source_ on iRecord website? We could possibly keep it hidden as _original location source_.
